nautobot/ipam/forms.py

```diff
@@ -237,7 +237,8 @@ class RIRFilterForm(BootstrapMixin, CustomFieldFilterForm):
 #
 
 
-class AggregateForm(BootstrapMixin, TenancyForm, PrefixFieldMixin, CustomFieldModelForm, RelationshipModelForm):
+# Important! PrefixFieldMixin must be last in the listing because it overrides save(commit=True)
```

Comment:
For all of these preceding commnts, docstrings instead please!

Reply:
As you pointed out, this is no longer true, so I'll revert it instead.